index.html

```diff
@@ -137,11 +163,27 @@ <h4>background-image</h4>
           </li>
           <li>
             <h4>controls</h4>
-            <P>Enables controls via mouse/touch when in flat view.</P>
+            <p>Enables controls via mouse/touch when in flat view.</p>
           </li>
           <li>
-            <h4>controls</h4>
-            <P>Enables controls via mouse/touch when in flat view.</P>
+            <h4>environment-image</h4>
+            <p>Controls the environmental reflection of the model. Normally if
+            background-image is set, that image will also be used for the
+            environment-image. Use environment-image to only set the reflection
+            without affecting the background.</p>
+          </li>
+          <li>
+            <h4>environment-intensity</h4>
+            <p>Controls the intensity of the environmental reflection. There is
+            always a reflection from the environment, even when background-image
```

Comment:
This is not true for KHR_materials_unlit models

Reply:
Will mention